cmd/completion.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
I think we could add the explanation over the command at the of the install section in the quick start. See: https://book.kubebuilder.io/quick-start.html

However, hidden such as:

<img width="648" alt="Screenshot 2020-03-20 at 01 37 30" src="https://user-images.githubusercontent.com/7708031/77129295-6357a080-6a4b-11ea-8e66-48ec36cdf6d7.png">


Something like: Click here to know how to configure the ....

Reply:
It is solved you can close.